disputer/disputer.js

```diff
@@ -15,6 +23,41 @@ class Disputer {
 
     // Instance of the expiring multiparty to perform on-chain disputes
     this.empContract = this.empClient.emp;
+
+    // Default config settings. Disputer deployer can override these settings by passing in new
+    // values via the `config` input object. The `isValid` property is a function that should be called
+    // before resetting any config settings. `isValid` must return a Boolean.
+    const defaultConfig = {
+      disputeDelay: {
+        // `disputeDelay`: Amount of time to wait after the request timestamp of the liquidation to be disputed.
+        // This makes the reading of the historical price more reliable. Denominated in seconds.
+        value: 60,
+        isValid: x => {
+          return x >= 0;
+        }
+      },
+      txnGasLimit: {
+        // `txnGasLimit`: Gas limit to set for sending on-chain transactions.
+        value: 9000000, // Can see recent averages here: https://etherscan.io/chart/gaslimit
+        isValid: x => {
+          return x >= 6000000 && x < 15000000;
+        }
+      }
```

Comment:
Same comment as [here](https://github.com/UMAprotocol/protocol/pull/1423#discussion_r424718980).

Reply:
Same as the other PR, feel free to ignore, I misread.